### ISSUE TITLE
Inconsistent style after using h.hsplit/h.group_rows to split images

### ISSUE THREAD

**Issue by AyinMostima** [Marsilea-viz/marsilea#42]
Requirement already satisfied: marsilea in d:\miniconda3\envs\dataanalysis\lib\site-packages (0.4.5)
Requirement already satisfied: legendkit in d:\miniconda3\envs\dataanalysis\lib\site-packages (from marsilea) (0.3.4)
Requirement already satisfied: matplotlib>=3.6 in d:\miniconda3\envs\dataanalysis\lib\site-packages (from marsilea) (3.9.2)
Requirement already satisfied: numpy in d:\miniconda3\envs\dataanalysis\lib\site-packages (from marsilea) (1.26.4)
Requirement already satisfied: pandas[parquet] in d:\miniconda3\envs\dataanalysis\lib\site-packages (from marsilea) (2.2.2)
Requirement already satisfied: platformdirs in d:\miniconda3\envs\dataanalysis\lib\site-packages (from marsilea) (4.2.2)
Requirement already satisfied: scipy in d:\miniconda3\envs\dataanalysis\lib\site-packages (from marsilea) (1.14.0)
Requirement already satisfied: seaborn in d:\miniconda3\envs\dataanalysis\lib\site-packages (from marsilea) (0.13.2)
Requirement already satisfied: contourpy>=1.0.1 in d:\miniconda3\envs\dataanalysis\lib\site-packages (from matplotlib>=3.6->marsilea) (1.2.1)
Requirement already satisfied: cycler>=0.10 in d:\miniconda3\envs\dataanalysis\lib\site-packages (from matplotlib>=3.6->marsilea) (0.12.1)
Requirement already satisfied: fonttools>=4.22.0 in d:\miniconda3\envs\dataanalysis\lib\site-packages (from matplotlib>=3.6->marsilea) (4.53.1)
Requirement already satisfied: kiwisolver>=1.3.1 in d:\miniconda3\envs\dataanalysis\lib\site-packages (from matplotlib>=3.6->marsilea) (1.4.5)
Requirement already satisfied: packaging>=20.0 in d:\miniconda3\envs\dataanalysis\lib\site-packages (from matplotlib>=3.6->marsilea) (24.1)
Requirement already satisfied: pillow>=8 in d:\miniconda3\envs\dataanalysis\lib\site-packages (from matplotlib>=3.6->marsilea) (10.4.0)
Requirement already satisfied: pyparsing>=2.3.1 in d:\miniconda3\envs\dataanalysis\lib\site-packages (from matplotlib>=3.6->marsilea) (3.1.2)
Requirement already satisfied: python-dateutil>=2.7 in d:\miniconda3\envs\dataanalysis\lib\site-packages (from matplotlib>=3.6->marsilea) (2.9.0)
Requirement already satisfied: pytz>=2020.1 in d:\miniconda3\envs\dataanalysis\lib\site-packages (from pandas[parquet]->marsilea) (2024.1)
Requirement already satisfied: tzdata>=2022.7 in d:\miniconda3\envs\dataanalysis\lib\site-packages (from pandas[parquet]->marsilea) (2024.1)
Requirement already satisfied: pyarrow>=10.0.1 in d:\miniconda3\envs\dataanalysis\lib\site-packages (from pandas[parquet]->marsilea) (16.1.0)
Requirement already satisfied: six>=1.5 in d:\miniconda3\envs\dataanalysis\lib\site-packages (from python-dateutil>=2.7->matplotlib>=3.6->marsilea) (1.16.0)

When I use this official code:
import matplotlib as mpl
import matplotlib.pyplot as plt
from matplotlib.colors import Normalize
import marsilea as ma
import marsilea.plotter as mp

from sklearn.preprocessing import normalize

pbmc3k = ma.load_data("pbmc3k")
exp = pbmc3k["exp"]
pct_cells = pbmc3k["pct_cells"]
count = pbmc3k["count"]

matrix = normalize(exp.to_numpy(), axis=0)

cell_cat = [
    "Lymphoid",
    "Myeloid",
    "Lymphoid",
    "Lymphoid",
    "Lymphoid",
    "Myeloid",
    "Myeloid",
    "Myeloid",
]
cell_names = [
    "CD4 T",
    "CD14\nMonocytes",
    "B",
    "CD8 T",
    "NK",
    "FCGR3A\nMonocytes",
    "Dendritic",
    "Megakaryocytes",
]

# Make plots
cells_proportion = mp.SizedMesh(
    pct_cells,
    size_norm=Normalize(vmin=0, vmax=100),
    color="none",
    edgecolor="#6E75A4",
    linewidth=2,
    sizes=(1, 600),
    size_legend_kws=dict(title="% of cells", show_at=[0.3, 0.5, 0.8, 1]),
)
mark_high = mp.MarkerMesh(matrix > 0.7, color="#DB4D6D", label="High")
cell_count = mp.Numbers(count["Value"], color="#fac858", label="Cell Count")
cell_exp = mp.Violin(
    exp, label="Expression", linewidth=0, color="#ee6666", density_norm="count"
)
cell_types = mp.Labels(cell_names, align="center")
gene_names = mp.Labels(exp.columns)

# Group plots together
h = ma.Heatmap(
    matrix, cmap="Greens", label="Normalized\nExpression", width=4.5, height=5.5
)
h.add_layer(cells_proportion)
h.add_layer(mark_high)
h.add_right(cell_count, pad=0.1, size=0.7)
h.add_top(cell_exp, pad=0.1, size=0.75, name="exp")
h.add_left(cell_types)
h.add_bottom(gene_names)

h.hsplit(labels=cell_cat, order=["Lymphoid", "Myeloid"])
h.add_left(mp.Chunk(["Lymphoid", "Myeloid"], ["#33A6B8", "#B481BB"]), pad=0.05)
h.add_dendrogram("left", colors=["#33A6B8", "#B481BB"])
h.add_dendrogram("bottom")
h.add_legends("right", align_stacks="center", align_legends="top", pad=0.2)
h.set_margin(0.2)
h.render()

I found that the styles of the upper and lower images on the right are inconsistent after segmentation.One has background and one does not.

![1724086274422](https://github.com/user-attachments/assets/5e38e300-6857-44a7-aefb-6e877a3232ea)

**Comment by Mr-Milk** [comment]
Thanks for using Marsilea. This is an interesting behavior. Are you running in some IDE environment like PyCharm dark mode? Because they will inject styles into matplotlib, which Marsilea has no control of.

**Comment by AyinMostima** [comment]
> Thanks for using Marsilea. This is an interesting behavior. Are you running in some IDE environment like PyCharm dark mode? Because they will inject styles into matplotlib, which Marsilea has no control of.感谢您使用Marsilea。这是一个有趣的行为。您是否在一些 IDE 环境中运行，例如 PyCharm 暗模式？因为他们会将样式注入到 matplotlib 中，而 Marsilea 无法控制。

No matter what renderer I use, whether it is inline, notebook or qt, when I set plt.style.use('seaborn-v0_8'), I need to
# Set global parameters
plt.rcParams['figure.facecolor'] = 'white' # Set the background of the figure to be transparent
plt.rcParams['axes.facecolor'] = 'white' # Set the background of the axes to be transparent
plt.rcParams['savefig.facecolor'] = 'white' # Save the image with transparent background
To remove all backgrounds, the top and bottom are consistent

**Comment by Mr-Milk** [comment]
This is not related to Marsilea. Marsilea doesn't modify any style settings. The styles are controlled by the user or the environment. If you don't want any background, you should set it to `'none'` instead of `'white'`, the latter is the default, see: https://matplotlib.org/stable/users/explain/customizing.html. If you still think this is an issue, please report it to matplotlib.
